sdk/identity/identity-cache-persistence/test/internal/node/msalNodeTestSetup.ts

```diff
@@ -0,0 +1,233 @@
+    await recorder.addSanitizers(
+      {
+        bodySanitizers: [
+          {
+            regex: true,
+            target: 'username=[^&"]+', // env sanitizers do not handle matching urlencoded params well (@ character is urlencoded)
+            value: "username=azure_username",
+          },
+          {
+            regex: true,
+            target: 'client_secret=[^&"]+',
+            value: "client_secret=azure_client_secret",
+          },
+          {
+            regex: true,
+            target: `client_assertion=[a-zA-Z0-9-._]*`,
+            value: "client_assertion=client_assertion",
+          },
+          {
+            regex: true,
+            target: 'device_code=[^&"]+',
+            value: "device_code=DEVICE_CODE",
+          },
+          {
+            regex: true,
+            target: `x-client-OS=[a-zA-Z0-9]+`,
+            value: `x-client-OS=Sanitized`,
+          },
+          {
+            regex: true,
+            target: `x-client-CPU=[a-zA-Z0-9]+`,
+            value: `x-client-CPU=Sanitized`,
+          },
+          {
+            regex: true,
+            target: `x-client-VER=[a-zA-Z0-9.-]+`,
+            value: `x-client-VER=identity-client-version`,
+          },
+          {
+            regex: true,
+            target: `client-request-id=[a-zA-Z0-9-]+`,
+            value: `client-request-id=${playbackValues.correlationId}`,
+          },
+        ],
+        bodyKeySanitizers: [
+          {
+            jsonPath: "$.device_code",
+            value: "DEVICE_CODE",
+          },
+          {
+            jsonPath: "$.bodyProvided.device_code",
+            value: "DEVICE_CODE",
+          },
+          {
+            jsonPath: "$.interval",
+            value: "0",
+          },
+          {
+            jsonPath: "$.client-request-id",
+            value: playbackValues.correlationId,
+          },
+          {
+            jsonPath: "$.access_token",
+            value: "access_token",
+          },
+          {
+            jsonPath: "$.bodyProvided.access_token",
+            value: "access_token",
+          },
+          {
+            jsonPath: "$.refresh_token",
+            value: "refresh_token",
+          },
+          {
+            jsonPath: "$.bodyProvided.refresh_token",
+            value: "refresh_token",
+          },
+          {
+            jsonPath: "$.id_token",
+            value:
+              "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6ImtpZCJ9.eyJhdWQiOiJhdWQiLCJpc3MiOiJodHRwczovL2xvZ2luLm1pY3Jvc29mdG9ubGluZS5jb20vMTIzNDU2NzgtMTIzNC0xMjM0LTEyMzQtMTIzNDU2Nzg5MDEyL3YyLjAiLCJpYXQiOjE2MTUzMzcxNjMsIm5iZiI6MTYxNTMzNzE2MywiZXhwIjoxNjE1MzQxMDYzLCJhaW8iOiJhaW8iLCJpZHAiOiJodHRwczovL3N0cy53aW5kb3dzLm5ldC9pZHAvIiwibmFtZSI6IkRhbmllbCBSb2Ryw61ndWV6Iiwib2lkIjoib2lkIiwicHJlZmVycmVkX3VzZXJuYW1lIjoiZGFucm9kcmlAbWljcm9zb2Z0LmNvbSIsInJoIjoicmguIiwic3ViIjoic3ViIiwidGlkIjoiMTIzNDU2NzgtMTIzNC0xMjM0LTEyMzQtMTIzNDU2Nzg5MDEyIiwidXRpIjoidXRpIiwidmVyIjoiMi4wIn0=.bm9faWRlYV93aGF0c190aGlz",
```

Comment:
is this a real token?

Reply:
nah, it's an exact copy of https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/test/node/msalNodeTestSetup.ts#L171-L181